scripts/scil_visualize_fodf.py

```diff
@@ -171,6 +175,13 @@ def _get_data_from_inputs(args):
     Load data given by args. Perform checks to ensure dimensions agree
     between the data for mask, background, peaks and fODF.
     """
+
+    def _check_mask_shape(_mask, _fodf):
```

Comment:
Use assert_same_resolution don't need extra function

Reply:
Done in f87598e. Note that I re-used what was being done for other arguments.